src/runtime/node/process.ts

```diff
@@ -73,6 +73,8 @@ export const {
   stdin,
   stdout,
   title,
+  throwDeprecation,
+  traceDeprecation,
```

Comment:
This two are tricky. Check node coverage + `require('process').throwDeprecation` is `undefined` in latest node.